tests/unit/__snapshots__/endpoint.spec.ts.snap

```diff
@@ -1,41 +1,61 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
-exports[`Endpoint > .getSchema() > should return the negative response schema 1`] = `
-{
-  "_type": "ZodObject",
-  "shape": {
-    "error": {
+exports[`Endpoint > .getResponses() > should return the negative responses 1`] = `
+[
+  {
+    "mimeTypes": [
+      "application/json",
+    ],
+    "schema": {
       "_type": "ZodObject",
       "shape": {
-        "message": {
-          "_type": "ZodString",
+        "error": {
+          "_type": "ZodObject",
+          "shape": {
+            "message": {
+              "_type": "ZodString",
```

Comment:
this, btw, is a matter of snapshot taking, could be added to vitest.setup.ts instead of helper

Reply:
addressed serialization in #2195